neuro_san/internals/graph/persistence/registry_manifest_restorer.py

```diff
@@ -65,18 +66,16 @@ def __init__(self, manifest_files: Union[str, List[str]] = None):
         self.logger = logging.getLogger(self.__class__.__name__)
 
     # pylint: disable=too-many-locals
-    def restore(self, file_reference: str = None):
+    def restore_from_files(self, file_references: Sequence[str]) -> Dict[str, AgentToolRegistry]:
         """
-        :param file_reference: The file reference to use when restoring.
-                Default is None, implying the file reference is up to the
-                implementation.
-        :return: an object from some persisted store
+        :param file_references: The sequence of file references to use when restoring.
+        :return: a built map of agents tools registries
         """
```

Comment:
Factor out shared functionality - restoring agents registries from sequence of file paths.

Reply:
FWIW: These days we are only really expecting a single manifest file, but I see why you have gone here -  it's the API you were given.